Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -63,4 +63,53 @@ Describe "Get-BqJob" {
     }
 }
 
+Describe "BqJob-Query" {
+
+    BeforeAll {
+        $r = Get-Random
+        $datasetName = "pshell_testing_$r"
+        $test_set = New-BqDataset $datasetName
+        $folder = Get-Location
+        $folder = $folder.ToString()
+        $filename = "$folder\classics.csv"
+        $table = New-BqTable -Dataset $test_Set "table_$r"
+        New-BqSchema -Name "Title" -Type "STRING" | New-BqSchema -Name "Author" -Type "STRING" |
+            New-BqSchema -Name "Year" -Type "INTEGER" | Set-BqSchema $table | 
+            Add-BqTabledata $filename CSV -SkipLeadingRows 1
+    }
+
+    It "should query out of a pre-loaded table" {
+        $job = Start-BqJob -Query "select * from $datasetName.table_$r where Year > 1900"
+        $job | Should Not Be $null
+        #TODO(ahandley): Add in checking the data once receive is done.
+    }
+
+    It "should query out of a pre-loaded table" {
+        $alt_tab = New-BqTable -Dataset $test_Set "table_res_$r"
+        $job = Start-BqJob -Query "select * from $datasetName.table_$r where Year > 1900" `
+            -DefaultDataset $test_set -DestinationTable $alt_tab -PollUntilComplete
+        $job = $job | Get-BqJob
+        $alt_tab = $alt_tab | Get-BqTable
+        $job.Status.State | Should Be "DONE"
+        $job.Configuration.Query.DefaultDataset.DatasetId | Should Be $test_set.DatasetReference.DatasetId
+        $job.Configuration.Query.DestinationTable.TableId | Should Be $alt_tab.TableReference.TableId
+        $alt_tab.NumRows | Should Be 2
+    }
+
+    It "should use legacy SQL when asked" {
+        $job = Start-BqJob -Query "select * from $datasetName.table_$r where Year > 1900" -UseLegacySql
+        $job.Configuration.Query.UseLegacySql | Should Be True
```

Comment:
Should have had a TODO in for that one too.  I need Receive- for that, and I'll add it in asap.